deploy/backplane/mtsre/openshift-connectors/10-mtsre-openshift-connectors-cr-admins.SubjectPermissions.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: managed.openshift.io/v1alpha1
```

Comment:
Can you rename this file to `10-mtsre-openshift-connectors-admins.SubjectPermissions.yml`

Reply:
done